docs/usage/secrets_rotation.md

```diff
@@ -47,6 +47,8 @@ The substituted Kubeconfig are provided after the initialized reconciliation was
 which were created with Gardener version `<= 0.28.0` used to have a Kubeconfig based on a client certificate instead of a static token.
 These client certificates are not revocable and thus a full credential rotation is not supported.
 
+You can check the `.status.credentials.rotation.kubeconfig` field in the `Shoot` to see when the rotation was last initiated or last completed.
```

Comment:
The `lastCompletionTime` will be initially set to the completion time of the first reconciliation with this new gardener version.
This will be confusing for endusers and "display wrong data", because the kubeconfig was never rotated.
I suggest to
- a) not set it initially and only after the first successful rotation OR
- b) document this behavior properly in order to avoid confusion

I slightly tend towards a).

Reply:
Good catch, I wasn't aware of this. I agree with a).

Reply:
My bad, I misread the if conditions... Everything is good here. Tested it as well :)